src/styles.css/card.css

```
@@ -0,0 +1,107 @@
+.page {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid var(--color-border);
+  border-radius: 5px;
+}
+
+.card__image {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.card__imac {
```

It's not always gonna be iMac :)
```suggestion
.card__image {
```

---

but whyyyyyy?:) I`m just in love with apple products) haha) 
__
I got u)